src/components/shapes/helpers.js

```diff
@@ -58,18 +58,24 @@ exports.extractPathCoords = function(path, paramsToUse) {
     return extractedCoordinates;
 };
 
-exports.getDataToPixel = function(gd, axis, isVertical) {
+exports.getDataToPixel = function(gd, axis, isVertical, opt) {
```

Comment:
Let's rename this `opt` argument to `refType`.

Reply:
fixed by 2d25857bbe5ee314f7da07669ca4158f072bc831